plugins/base/src/test/kotlin/renderers/html/HeaderTest.kt

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package renderers.html
+
+import org.jetbrains.dokka.SourceLinkDefinitionImpl
+import org.jetbrains.dokka.base.renderers.html.HtmlRenderer
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import renderers.testPage
+import java.net.URL
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class HeaderTest : HtmlRenderingOnlyTestBase() {
```

Comment:
Oh, I even forgot we have `HtmlRenderingOnlyTestBase`... I personally try to stick to `testInline` (see `SignatureTest` for an example), it can verify all stages and levels (from documentable to html), and I think its tests are more correct and will be easier to refactor down the road. 

Once dokka-base is broken down into more manageable modules, I'll be advocating for cutting down on the number of test DSLs, so that there's one DSL per module or at least one per level, and it should be clear which one to use in which scenario (unlike now).

So these tests can stay the way that they are (as long as they work and find bugs) and we'll address it later on a larger scale, or you can give `testInline` a go, up to you :)

Reply:
Okay, agree, I will refactor those new tests to use `BaseAbstractTest`